File: assignment_3/assi_3_4.java
import java.util.*;
public class assi_3_4
{
    public static boolean prime(int x)
    {
        for(int i=2; i*i<=x; i++)
        {
            if(x%i==0)
            {
                return true;
            }
        }
        return false;
    }
    public static Scanner scn= new Scanner (System.in);
    public static void main(String [] args)
    {
        int x=scn.nextInt();
        boolean res=prime(x);
        if(res==false)
        {
            System.out.println("True");
        }
        else
        {
            System.out.println("False");
        }
    }
}

File: assignment_3/assi_3_4.cpp
#include<iostream>
using namespace std;
bool prime(int n)
{
    for(int i=2; i*i<=n; i++)
    {
        if(n%i==0)
        {
            return true;
        }
    }
    return false;
}
int main()
{
    int n;
    cin>>n;
    bool res=prime(n);
    if(res==0)
    {
        cout<<"True"<<endl;
    }
    else 
    {
        cout<<"False"<<endl;
    }
}